test/integration/transformation_filter_integration_test.cc

```diff
@@ -58,6 +58,16 @@ const std::string BODY_TRANSFORMATION =
         text: "{{abc}}"
 )EOF";
 
+const std::string BODY_TRANSFORMATION_METADATA =
+    R"EOF(
+  clear_route_cache: true
```

Comment:
i would mention here and in the proto-docs that `clear_route_cache` does nothing for upstream filters, and is only set here to test that envoy behaves well even if it is set